android/src/main/java/com/revenuecat/purchases/react/RNPurchasesModule.java

```diff
@@ -276,9 +278,10 @@ public void getCustomerInfo(final Promise promise) {
     }
 
     @ReactMethod
+    @Deprecated // Use setPurchasesAreCompletedBy instead
     public void setFinishTransactions(boolean enabled) {
```

Comment:
I've marked this function as deprecated but would be okay with removing it if we want to be more strict about the observer mode -> purchasesAreCompletedBy migration

Reply:
I believe if it still works, we should try to avoid removing functions if possible in order to avoid breaking changes. Having said that, since this is part of android, which is not part of the react-native-purchases public API normally, I think we can remove this and have the TS `setFinishTransactions` function call the new method instead. Wdyt?

Reply:
+1 to removing this, it's private, nobody should be calling it manually.

Reply:
Sounds good, I'll remove this both here and in `purchases.ts` 👍

Reply:
`setPurchasesAreCompletedBy()` isn't currently publicly available in `purchases.ts` - is there any reason for it to be public? Typically I think of this setting as something that you set when calling `configure()` and then don't modify during runtime